api/v1beta1/managedclustermodule_types.go

```diff
@@ -28,6 +28,9 @@ type ManagedClusterModuleSpec struct {
 	// Namespace describes the Spoke namespace, in which the ModuleSpec should be applied.
 	Namespace string `json:"namespace,omitempty"`
 
+	// JobNamespace describes the Hub namespace, to which Build and Sign Jobs should be deployed.
+	JobNamespace string `json:"jobNamespace,omitempty"`
```

Comment:
Now that we have this, should we change the `Namespace` field to `SpokeNamespace`? Your comment is excellent but I always prefer self documented code :)